pkg/kancontroller/kancontroller.go

```diff
@@ -112,3 +118,12 @@ func isCACertMounted() bool {
 
 	return true
 }
+
+func createOrUpdateCRDs() bool {
+	createOrUpdateCRD := os.Getenv(createOrUpdateCRDEnvVar)
+	if createOrUpdateCRD == "" {
+		return false
+	}
+
+	return createOrUpdateCRD == "true"
```

Comment:
TIOLI: what about `True`, `TRUE`, `t`, `T`? Feels like the previous approach of using `strconv.ParseBool()` is more robust.

Reply:
I agree, I am also inclined to change it back to what it was.